.github/workflows/application.yml

```diff
@@ -32,7 +32,7 @@ jobs:
             fail-fast: false
             matrix:
                 php: ["7.4", "8.0", "8.1"]
-                symfony: ["^4.4", "5.2.*", "5.3.*", "5.4.*"]
+                symfony: ["^4.4", "5.3.*", "5.4.*"]
```

Comment:
Same as [here](https://github.com/Sylius/Sylius/pull/13829#discussion_r844059207), can we, please, explain this change in the PR description?